opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -432,7 +433,7 @@ std::shared_ptr< RawKeyword > createRawKeyword( const string_view& kw, ParserSta
 
     std::string msg = "Expected the kewyord: " + sizeKeyword.first
                     + " to infer the number of records in: " + keywordString;
-
+    parserState.deck->getMessageContainer().error(msg);
```

Comment:
Default setting for ``ParseContext::PARSE_UNKNOWN_KEYWORD`` is ``throw``, so I guess this is an error.